spec/classes/init_spec.rb

```diff
@@ -558,8 +581,9 @@
   end
 
   context 'with host_entries containing post for fqdn' do
-    let(:facts) { { :fqdn => 'myhost.example.com',
-                    :ip   => '10.0.0.5',} }
+    let(:facts) { { :fqdn      => 'myhost.example.com',
+                    :ip        => '10.0.0.5',
+                    :ipaddress => '10.1.2.3',} }
```

Comment:
The same as above. If neither fqdn_ip_address nor ipaddress are set, this test fails.

How do you think this can be avoided?